### DISCUSSION TITLE
Limit the charging power

### DISCUSSION THREAD

**Discussion by Moorole** [seerge/g-helper#1472] [Ideas]
I wa just wondering if, on top of limiting the level the battery is charged to, it is possible to limit the wattage which is used to fill the battery to prevent damage to the battery by overheating. Don't know if this is an easy thing to implement or if it requires deep interaction with the system. Thank you for all the great work and a happy sunday to everyone.

**Comment by seerge** [comment]
@Moorole no, that's not possible. Asus have provided only one (and very simple) endpoint, where you can set value from 0 to 100% which would be a charge limit , and the followed precisely (or not really) by hardware :)
